sdk/servicebus/Azure.Messaging.ServiceBus/tests/SessionRecords/ServiceBusManagementClientLiveTests/CorrelationFilterPropertiesAsync.json

```diff
@@ -1,16 +1,16 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://jolovservicebus.servicebus.windows.net/006bafbe?api-version=2017-04",
+      "RequestUri": "https://omqsquire.servicebus.windows.net/006bafbe?api-version=2017-04",
       "RequestMethod": "PUT",
       "RequestHeaders": {
         "Authorization": "Sanitized",
         "Content-Length": "843",
         "Content-Type": "application/atom\u002Bxml",
```

Comment:
What is the reasoning behind re-recording all of these? I would expect we may have new live tests but existing ones should pass playback without re-recording.

Reply:
![](https://media3.giphy.com/media/KEh5kliRTSVJm/giphy.gif)

I haven't got a good grasp on the recording framework.   I had fails.  I set the mode to Record.  I added whatever got spit out.    No objection to changing, but how do?

Reply:
Can you include what the failures were?

Reply:
Not sure that I understand what you're looking for, but I was getting flagged for not having recordings for the Sas-based tests.